frontend/src/TaskDetail.svelte

```diff
@@ -198,6 +198,11 @@ async function load() {
   files = json['sources'].map((source) => new SourceFile(source));
   summaryComments = json['summary_comments'];
 
+  // Append summary to summary comments
```

Comment:
Looks like this is outdated? The summary comments should already be in `summary_comments`.